modules/tinymce/src/plugins/table/test/ts/browser/InsertColumnTableResizeTest.ts

```diff
@@ -1,200 +1,455 @@
-import { Assertions, Chain, Guard, Log, NamedChain, TestLogs, UiFinder } from '@ephox/agar';
-import { Assert, UnitTest } from '@ephox/bedrock-client';
-import { McEditor, TinyDom } from '@ephox/mcagar';
-import { SelectorFind } from '@ephox/sugar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyDom, TinyHooks } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
+import { SelectorFind, Width } from '@ephox/sugar';
+import { assert } from 'chai';
 
+import Editor from 'tinymce/core/api/Editor';
 import Plugin from 'tinymce/plugins/table/Plugin';
```

Comment:
This needs a `Theme` import as well so it's properly initialized and doesn't use a prebuilt version (ie from the last time you ran `yarn dev`). This is needed for every test.